src/api/apiconfig_api/v1/_base_policy.xml

```diff
@@ -12,15 +12,23 @@
 -->
 <policies>
     <inbound>
-        <base />
+        <base/>
+        <set-variable name="isGet" value="@(context.Request.Method.Equals("GET"))" />
+        <choose>
+            <when condition="@(context.User.Groups.Any(g => g.Name == "Read Only") && !context.Variables.GetValueOrDefault<bool>("isGet"))">
```

Comment:
where is defined [read only group](https://portal.azure.com/?l=en.en-us#blade/Microsoft_Azure_ApiManagement/GroupBlade/groupOverview/id/%2Fsubscriptions%2Fbbe47ad4-08b3-4925-94c5-1278e5819b86%2FresourceGroups%2Fpagopa-d-api-rg%2Fproviders%2FMicrosoft.ApiManagement%2Fservice%2Fpagopa-d-apim%2Fgroups%2Fread-only/name/Read%20Only) ?

Reply:
added terreform definition for [read-only group ](https://github.com/pagopa/pagopa-infra/pull/64/files#diff-436422a5916b2158974132848de0a192bc0e9df0b06d9d0f776594540ac9f7cdR73-R80)